doc/files/package-lock.json.md

```diff
@@ -46,6 +46,9 @@ The version of the package this is a package-lock for. This must match what's in
 An integer version, starting at `1` with the version number of this document
 whose semantics were used when generating this `package-lock.json`.
 
+### requires
```

Comment:
I'm not sure that this is accurate, here's what the package-lock.json file looks like for me (generated using 5.4.2):

```patch
     "@types/supertest": {
       "version": "2.0.3",
       "resolved": "https://registry.npmjs.org/@types/supertest/-/supertest-2.0.3.tgz",
       "integrity": "sha512-QxgjuDhQEq4tPJXTOnGoN4P/BWwvftvkaCZhXMp1C1Otk9kGXb7l/wA7kwz3MwtEMnovdYikS+ZtvaQ4RbhF4g==",
-      "dev": true
+      "dev": true,
+      "requires": {
+        "@types/superagent": "3.5.5"
+      }
     },
```

Reply:
My apologies, it looks like it's specified in the file in two different places; at the top level, it's a boolean, inside of a package descriptor, it's an object. Not sure the best way to handle that.

Reply:
So how should we handle this?  Some docs are better than none... and if this isn't to be accepted I'll just close it.